package/MDAnalysis/analysis/msd.py

```diff
@@ -288,9 +290,12 @@ def __init__(self, u, select, msd_type='xyz', fft=True, **kwargs):
             Defaults to ``True``.    
 
         """
+        if isinstance(u, groups.UpdatingAtomGroup):
```

Comment:
Nearly forgot, please do add a test for this. Might as well try to keep test coverage for MSD at 100% :)